python/hsfs/core/feature_monitoring_result.py

```diff
@@ -0,0 +1,122 @@
+#
+#   Copyright 2023 Hopsworks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import humps
+from typing import Any, Dict, List, Optional
+from hsfs import util
+
+
+class FeatureMonitoringResult:
+    def __init__(
+        self,
+        feature_store_id: int,
+        entity_id: int,
```

Comment:
I think we can skip `entity_id`. Please see my [comment here](https://github.com/logicalclocks/hopsworks-ee/pull/1267/files#r1115999003).